internal/v5/auth.go

```diff
@@ -21,10 +20,13 @@ import (
 )
 
 const (
-	PromulgatorsGroup      = "charmers"
-	opAccessCharmWithTerms = "access-operation"
-	opOther                = "other-operation"
-	defaultMacaroonExpiry  = 24 * time.Hour
+	PromulgatorsGroup = "charmers"
+	// opAccessCharmWitTerms indicates an operation of accessing the archive of
+	// a charm that requires agreement to certain terms and conditions.
+	opAccessCharmWithTerms = "op-get-with-terms"
+	// opOther indicated all other operations.
```

Comment:
s/indicated/indicates/

    // This operation should not be added as part of a macaroon caveat.

?